ansible/manifests/qlean-deployment.yml.j2

```diff
@@ -0,0 +1,92 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ node_name | replace('_', '-') }}
+  namespace: {{ k8s_namespace }}
+  labels:
+    app: {{ node_name | replace('_', '-') }}
+    managed-by: ansible
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ node_name | replace('_', '-') }}
+  template:
+    metadata:
+      labels:
+        app: {{ node_name | replace('_', '-') }}
+    spec:
+      containers:
+      - name: qlean
+        image: {{ qlean_docker_image }}
+        imagePullPolicy: IfNotPresent
+        args:
+          - --genesis /config/config.yaml
+          - --validator-registry-path /config/validators.yaml
+          - --bootnodes /config/nodes.yaml
+          - --data-dir /data
+          - --node-id {{ node_name }}
+          - --node-key /config/{{ node_name }}.key
+          - --listen-addr /ip4/0.0.0.0/udp/{{ quic_port }}/quic-v1
+          - --prometheus_port {{ metrics_port }}
```

Comment:
we need to find out a way to put ansible, local and k8 args in the same place (different sections may be) so that a client team doesn't need to hunt different places for the args

it can be done as files in a client subdirectory may be

Reply:
I'll handle this in a separate PR when I make the k8s changes.